sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -226,6 +226,15 @@ export class MessageSender extends LinkEntity<AwaitableSender> {
         throw translateServiceBusError(e);
       }
 
+      if (!this.isOpen()) {
```

Comment:
I am not sure how expensive this call is but feels like an overhead for every send call. Should we wrap send in a try catch instead?

Reply:
We have the same check at the beginning as well.  I will change to just do undefined check, as that's probably what we care most for this issue.  We also have a similar check in `getMaxMessageSize()` and throw error if `link` is undefined.